src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/judgeapproval/HearingProcessor.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.judgeapproval;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DynamicList;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.HearingInstructionProcessable;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.YesOrNo;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.judgeapproval.AnotherHearingRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.judgeapproval.JudgeApproval;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.review.DraftOrderDocReviewCollection;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.review.PsaDocReviewCollection;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.wrapper.DraftOrdersWrapper;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.lang.String.format;
+import static java.util.Optional.ofNullable;
+
+@Component
+class HearingProcessor {
```

Comment:
This is a good example of a class/method that could do with javadocs. It's almost impossible to know what is going on without reading the implementation of the methods

Reply:
[e032596](https://github.com/hmcts/finrem-case-orchestration-service/pull/1997/commits/e032596adbc8224bf8b416ece1089447a1bda715)